### PR TITLE
Feature/7646 certificates detail screens/storybranch

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description

For test and vaccination detail screens

1. Design changes
2. Title adjustments
3. Label adjustments and changes
4. Adding stars to gradient
5. Adjusting date formats
6. key value additions
 
## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7756
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7757

## Screenshots
Simulator Screen Shot - iPhone 12 - 2021-06-15 at 19 12 10
Simulator Screen Shot - iPhone 12 - 2021-06-15 at 19 20 58
Simulator Screen Shot - iPhone 12 - 2021-06-15 at 19 18 51

Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-06-16 at 16 58 39
Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-06-16 at 16 58 43
Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-06-16 at 16 58 45

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/Checkin/EditCheckInTime/Views/GradientBackgroundView.swift [hunk 1]
```diff
@@ -49,25 +51,43 @@ class GradientBackgroundView: UIView {
 		gradientView.translatesAutoresizingMaskIntoConstraints = false
 		addSubview(gradientView)
 		gradientView.type = type
+		
+		let starsView = UIImageView(image: UIImage(named: "stars"))
+		if displayStars {
+			starsView.translatesAutoresizingMaskIntoConstraints = false
+			gradientView.addSubview(starsView)
+		}
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/Checkin/EditCheckInTime/Views/GradientBackgroundView.swift:59]: adding the stars in this view will not allow all gradients to use them
As discussed earlier please add the stars to `GradientView` so all other can use it too.
Extend cases with fe `blueWihStars` you can also add the subview with images there.

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateDetailsQRCodeCell.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import UIKit
+
+class HealthCertificateDetailsQRCodeCell: UITableViewCell, ReuseIdentifierProviding {
+
+	// MARK: - Init
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setupView()
+		isAccessibilityElement = false
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/HealthCertificateDetailsQRCodeCell.swift:14]: Maybe move all of this to `setupView() `?

File: src/xcode/ENA/ENA/Source/Views/GradientView.swift [hunk 3]
```diff
@@ -34,16 +34,56 @@ class GradientView: UIView {
...
+	private var withStars: Bool = false
+	private var starImageView: UIImageView = UIImageView()
+
+	private func setupStars() {
+		guard let image = type.starImage else {
+			starImageView.removeFromSuperview()
+			return
+		}
+
+		starImageView = UIImageView(image: image)
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Views/GradientView.swift:68]: we need to make sure UIImageView is nil before set an add it. If it's not nil there might be an old UIImageView, better remove that one from superview before assigning a new one.

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

**Review by KaiTeuber — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**